bluepyopt/deapext/algorithms.py

```diff
@@ -186,7 +183,8 @@ def eaAlphaMuPlusLambdaCheckpoint(
                       halloffame=halloffame,
                       history=history,
                       logbook=logbook,
-                      rndstate=random.getstate())
+                      rndstate=random.getstate(),
+                      param_names=param_names)
```

Comment:
So, for this comment and all the other comments about param_names: we took the decision to add the parameter names to the checkpoints because it is the only way to know the order of the parameters. If we loose the configuration files or if the order of the parameters are changed in the configuration file, it will create a discrepancy between the order of the parameters used to write the checkpoints and the one used to read the checkpoint (such a bug will be extremely difficult to track down).

Reply:
Alright, thanks for the explanation.